spec/defines/topic_spec.rb

```diff
@@ -25,6 +25,39 @@
         }
       end
 
+      context 'when create topic demo for kafka v3' do
+        let(:title) { 'demo' }
+        let :params do
+          {
+            'ensure'             => 'present',
+            'bootstrap_server'   => 'localhost:9092',
+            'replication_factor' => 1,
+            'partitions'         => 1
+          }
+        end
+
+        it {
+          is_expected.to contain_exec('create topic demo').with(
+            command: 'kafka-topics.sh --create --bootstrap-server localhost:9092 --replication-factor 1 --partitions 1 --topic demo '
+          )
+        }
+      end
+
+      context 'when create topic without zookeeper or bootstrap_server' do
+        let(:title) { 'demo' }
+        let :params do
+          {
+            'ensure'             => 'present',
+            'replication_factor' => 1,
+            'partitions'         => 1
+          }
+        end
+
+        it {
+          is_expected.to compile.and_raise_error('Either zookeeper or bootstrap_server parameter must be defined!')
```

Comment:
Maybe wrapping this in a regex will fix the spec error

Reply:
I did this before, but misinterpreted the linting error regarding the `%r{}`. Should be fine now!